### PR TITLE
feat(performance): Add guide anchor for team key transactions

### PR DESCRIPTION
**Pull request by Zylphrex**
This adds in an introduction for the new team key transactions as wells as to
let existing users know that if they previously had key transactions, they'll
have to migrate them over by re-keying them for their teams.

# Screenshots

## User did not have Key Transactions

image

## User did have Key Transactions

image

image

### CODE REVIEW

File: src/sentry/discover/endpoints/discover_key_transactions.py [hunk 1]
```diff
@@ -24,6 +24,32 @@ class KeyTransactionPermission(OrganizationPermission):
     }
 
 
+class LegacyKeyTransactionCountEndpoint(KeyTransactionBase):
```
Comment (wmak) [inline src/sentry/discover/endpoints/discover_key_transactions.py:27]: to check, this is to make EA easier right? my initial reaction is we could just re-use the existing `KeyTransactionEndpoint` but I might be missing something here.

**Review by wmak — APPROVED** [review]
question about the new endpoint, but LGTM regardless

File: src/sentry/discover/endpoints/discover_key_transactions.py (hunk 1 above)
Comment (Zylphrex) [reply src/sentry/discover/endpoints/discover_key_transactions.py:27]: Unsure how we would use the existing `KeyTransactionEndpoint` for this.

To clarify, this endpoint should be temporary during the rollout of team key transactions to make users that previously had any key transactions (not just a particular key transaction) aware that their old key transactions are gone.

Comment (wmak) [reply src/sentry/discover/endpoints/discover_key_transactions.py:27]: My mistake, thought we had an endpoint that already returns the count of KT. :+1: that explanation of the endpoint makes sense

**Review by dashed — APPROVED** [review]

**Pull request merged**